hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/RestBase.java

```diff
@@ -33,13 +41,27 @@
     @AutoTenant
     protected Inventory inventory;
 
+    // using the @AllPermissive annotation the access will be always granted
     @Inject
+    @AllPermissive
```

Comment:
This can't go into master. I understand you left it on while you're working on this but we need to remember to switch to authz'ed when you're done.